tensilelite/Tensile/Components/LSU.py

```diff
@@ -245,7 +242,10 @@ def writeReadReduction(self, writer, kernel):
                 DSLoadBX     = DSLoadB32
                 numInstPerVW = bytesPerVector // 4
                 regsPerStore = 1
-
+            maxOffset = (kernel["LocalSplitU"] -1) * ldsStride + ((numVgprPerLSU // self.LSUfullVw -1) * (numInstPerVW-1) + numInstPerVW) * regsPerStore * (bpr * kernel["WavefrontSize"])
```

Comment:
This is still incorrect.
Please carefully check the original calculation.
                        regIdx = (i * numInstPerVW + v) * regsPerStore
                        offset = r * ldsStride + regIdx * (bpr * kernel["WavefrontSize"])

v part is incorrect. You changed wrong location.